app/src/main/res/layout/dialog_team_mode.xml

```diff
@@ -75,5 +75,12 @@
         tools:listitem="@layout/cell_team_mode_color_circle_select"
         tools:visibility="visible" />
 
+    <TextView
```

Comment:
Why is this an extra text block that is always shown at the very bottom?

To save space, I'd rather …

1. … append it to the text at the top:
	> When mapping the same area together in a group, you can enter team mode to split up quests between all team members. Every team member needs to enable it on their own device. Remember to disable it after shared mapping!
2. … leave it at the bottom, but only show it when no team size is entered yet.
3. … leave it at the bottom, but only show it after a color is selected.

Reply:
I think option "1 + it hides together with the text at the top after the team size has been entered" or option 3 is best.

Reply:
@matkoniecz What's the current state of this PR?

Reply:
> Why is this an extra text block that is always shown at the very bottom?

As it is an extra info that is explaining use, so putting it at the end seemed logical.

> hides together with the text at the top after the team size has been entered

Given situation that triggered this PR - it is likely that it would be missed. Also, someone processing it sequentially: read message on top, answer question would never read it.

> … leave it at the bottom, but only show it after a color is selected.

I will implement this one.

> @matkoniecz What's the current state of this PR?

I am trying to finish it right now to close started, but not completed projects.

Reply:
As I worried - it works poorly for large team sizes.
![screen05](https://user-images.githubusercontent.com/899988/128853796-52c8faa6-7159-4ae1-b8b3-cc2483de9a87.png)